Testing/Temporary/CTestCostData.txt

```diff
@@ -0,0 +1 @@
+---
```

Comment:
Looks like this file accidentally slipped into the commit.

It has also [happened in the past](https://github.com/InsightSoftwareConsortium/ITK/commit/085907f8cc487e28925e2c1469ccbd4196242a9a#diff-61b29af89ba6a8e5ca026504561932db).

Reply:
Removed and .gitignore added.